pages.es/linux/zypper.md

```diff
@@ -0,0 +1,27 @@
+# zypper
+
+> Utilidad para la gestión de paquetes en SUSE y openSUSE.
+
```

Comment:
We need a more information link.

Reply:
added link
(should I add the link in zypper English version?)

Reply:
@victorhck Yep.

Reply:
> @victorhck Yep.

done!